umm-spec-lib/src/cmr/umm_spec/migration/version/interface.clj

```diff
@@ -0,0 +1,22 @@
+(ns cmr.umm-spec.migration.version.interface
```

Comment:
I think we can remove this namespace and add the two functions in `cmr.umm-spec.migration.version.core`.